Header header
float32[] ref_traj
float32 ref_size
float32 Ts


# This is a trajectory that is transformed from a certain frame to the rover frame at time 0 in 
# the following sense: x, y, psi start positions are 0,0,0.
# x positive is strait ahead, y positive is to the right, psi starts positive clockwise from x axis.



